ui/index.d.ts

```diff
@@ -317,6 +309,13 @@ export declare abstract class AzureAccountTreeItemBase extends AzExtParentTreeIt
      */
     public constructor(parent?: AzExtParentTreeItem, testAccount?: TestAzureAccount);
 
+    //#region Methods implemented by base class
```

Comment:
I'm find with most of your changes to this file, but this one screwed up the regions

Reply:
Sorry, I think it was just the automatic format on save for tslint